detail/node.vue

```diff
@@ -145,6 +198,15 @@ export default {
       return isOk
         ? 'success'
         : 'error';
+    },
+    async loadMetrics() {
+      const schema = this.$store.getters['cluster/schemaFor'](METRIC.NODE);
+
+      if (schema) {
+        const metrics = (await this.$store.dispatch('cluster/find', { type: METRIC.NODE, id: this.value.id }));
```

Comment:
This needs: `opt: {force: true}` or you're just going to get back the cached object from the store after the time and no new API call.

Reply:
Done.